packages/cozy-client/src/Schema.js

```diff
@@ -1,6 +1,9 @@
 import keyBy from 'lodash/keyBy'
+import keys from 'lodash/keys'
```

Comment:
What's the difference with `Object.keys`? Why do we specially need to use a lodash method instead of the native method which does [the same thing](https://github.com/lodash/lodash/blob/master/keys.js#L34)? I don't think we need to handle array likes here which is the only difference IMHO

Reply:
Readability

Reply:
Object.keys seems to be the way to go here :)

I don't know what you mean by "readability" here, I guess it's because "Object." is not there. In this case `const keys = Object.keys` does the job.